docs/documentation/platform/dynamic-secrets/mongo-db.mdx

```diff
@@ -0,0 +1,117 @@
+---
+title: "Mongo DB"
+description: "Learn how to dynamically generate Mongo DB Database user credentials."
+---
+
+The Infisical Mongo DB dynamic secret allows you to generate Mongo DB Database credentials on demand based on configured role.
+
+
+<Info>
+				If your using Mongo Atlas, please use [Atlas Dynamic Secret](./mongo-atlas) as MongoDB commands are not supported by atlas.
+</Info>
+
+## Prerequisite
+Create a user with the required permission in your SQL instance. This user will be used to create new accounts on-demand.
```

Comment:
> Create a user with the required permission in your SQL instance.

Seems to be from another Dynamic Secret doc page

Reply:
Updated